distribution/lib/Standard/Database/0.0.0-dev/src/Main.enso

```diff
@@ -5,20 +5,30 @@ import project.Connection.Client_Certificate.Client_Certificate
 import project.Connection.SSL_Mode.SSL_Mode
 import project.Connection.Connection_Options.Connection_Options
 
-import project.Connection.Postgres_Options
-import project.Connection.SQLite_Options
-import project.Connection.Redshift_Options
+import project.Connection.Postgres_Options.Postgres_Options
+import project.Connection.Redshift_Options.Redshift_Options
+import project.Connection.Redshift_Options.AWS_Credential
+import project.Connection.SQLite_Options.SQLite_Options
+import project.Connection.SQLite_Options.In_Memory
 import project.Data.SQL_Query.SQL_Query
 
 export project.Connection.Credentials.Credentials
 
 export project.Connection.SSL_Mode.SSL_Mode
 export project.Connection.Client_Certificate.Client_Certificate
 export project.Connection.Connection_Options.Connection_Options
+export project.Connection.Postgres_Options.Postgres_Options
+export project.Connection.Redshift_Options.Redshift_Options
+export project.Connection.Redshift_Options.AWS_Credential
+export project.Connection.SQLite_Options.SQLite_Options
+export project.Connection.SQLite_Options.In_Memory
 
 export project.Connection.Database
 export project.Data.SQL_Query.SQL_Query
 
-from project.Connection.Postgres_Options export Postgres_Options, Postgres
-from project.Connection.SQLite_Options export SQLite_Options, In_Memory, SQLite
-from project.Connection.Redshift_Options export Redshift_Options, Redshift, AWS_Credential
+from project.Connection.Postgres_Options.Postgres_Options import Postgres
+from project.Connection.Redshift_Options.Redshift_Options import Redshift
+from project.Connection.SQLite_Options.SQLite_Options import SQLite
```

Comment:
Don't we want to have _all_ imports before _all_ exports?

Otherwise, I'd at least add a newline between the two, as otherwise it's harder to distinguish the two groups.